### DISCUSSION TITLE
clr-wizard--inline has been removed but it is necessary for vSphere Plugin development

### DISCUSSION THREAD

**Discussion by vpatelpstg** [vmware-clarity/ng-clarity#1021] [Q&A] [answered]
I have been using "clr-wizard--inline" property with "clr-wizard" in version 13. I need to upgrade our angular and Clarity to the latest versions, but I can see that the property has been removed from version 15 onwards by changes in [this commit](https://github.com/vmware-clarity/ng-clarity/commit/7470769480b843dc332adbd25f67624ce4979e56).

I use Clarity to develop a vSphere Plugin so that the plugin GUI matches the overall GUI of vSphere. One common use case in a vSphere plugin is displaying contents in a modal that is presented by the host vSphere GUI. The plugin is allowed to display contents inside an iFrame inside the body of this modal. If a plugin developer wants to show a wizard inside this kind of view, an inline wizard is the only option. Otherwise the user sees a modal inside a modal, and it simply does not work correctly.

This is how the wizard looks with the "clr-wizard--inline" property in Clarity 13.
![Screenshot 2023-10-27 at 12 22 03 PM](https://github.com/vmware-clarity/ng-clarity/assets/111947379/8849aea1-7add-419a-9e2d-4e98a11734ae)

And this is after the update to Clarity 16.
![Screenshot 2023-10-27 at 12 21 42 PM](https://github.com/vmware-clarity/ng-clarity/assets/111947379/3a679f97-6ebd-4dd4-a661-0c7e36cdd902)

Does Clarity 15 have some other mechanism to properly display a wizard inside a vSphere Plugin Modal? If not, can the clr-wizard--inline feature please be re-implemented? We are happy to do the work and submit a PR.

**Comment by vpatelpstg** [comment]
Tagging @kevinbuhmann for help, because he authored the commit mentioned above.

**Comment by LynxDelli** [comment]
@vpatelpstg Please reach us via VMware-Slack to channel clarity-support.

**Reply by royling** [reply]
@vpatelpstg any conclusion on this? did you get a solution?

**Reply by kevinbuhmann** [reply, marked as answer]
The inline wizard has been added back in v15.13.0 and v16.1.0.

https://github.com/vmware-clarity/ng-clarity/pull/1067

https://github.com/vmware-clarity/ng-clarity/pull/1068

**Reply by ngenereux-ps** [reply]
@kevinbuhmann was this also ported to v17.x? We just tried upgrading form 16 to 17 and our wizards broke again. Would you like me to log a bug?

**Reply by kevinbuhmann** [reply]
Sorry about that, I think we were working on v17 on a beta branch at the time and I didn't port this over. I'll take care of that.

**Reply by ngenereux-ps** [reply]
Thanks so much, we appreciate it!
